zsl/interface/gearman/task_filler.py

```diff
@@ -7,32 +7,27 @@
 from __future__ import print_function
 from __future__ import unicode_literals
 
-# Initialize app
-from zsl.interface.gearman import importer
-importer.append_asl_path_to_pythonpath()
-from zsl.interface.importer import initialize_cli_application, InitializationContext
-initialize_cli_application(InitializationContext(unit_test=False))
 
-import sys
 import gearman
 import json
-from zsl.application.service_application import service_application as app
 
-if __name__ == "__main__":
-    if len(sys.argv) != 3:
-        sys.stderr.write("Usage: task_filler.py <task name> <task data (file:// for file input, otherwise raw data)>\n")
-        exit(1)
+from zsl import Config, inject, Injected
 
-    print("Initializing client.")
```

Comment:
I'd still use logging here and used maybe INFO message level. It's quite common. There are also more messages like that.